en/docs/learn/export-user-info-to-hubspot.md

```diff
@@ -0,0 +1,37 @@
+#Exporting WSO2 Identity Server User Information to HubSpot
+
+
+HubSpot is a customer relationship management solution that brings companies and customers together. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the 
+Hub spot platform. 
+
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+    
+##Try it Out
+
+1. Login into [HubSpot](https://app.hubspot.com/) and got to [contact](https://app.hubspot.com/contacts) section.
+2. Click on **Import** button in the dashboard.
+![](../assets/img/tutorials/hubspot-provisioning-user/step-2.png)
+3. In the next window select **Import**.
+![](../assets/img/tutorials/hubspot-provisioning-user/step-3.png)
+4. Then select File from computer. 
+![](../assets/img/tutorials/hubspot-provisioning-user/step-4.png)
+5. In TYPE section select One file.
+![](../assets/img/tutorials/hubspot-provisioning-user/step-5.png)
+6. The select One object.
+![](../assets/img/tutorials/hubspot-provisioning-user/step-6.png)
+7. Then select Contacts.
+![](../assets/img/tutorials/hubspot-provisioning-user/step-7.png)
+8. Then upload `users.csv`.
+![](../assets/img/tutorials/hubspot-provisioning-user/step-8.png)
+9. Perform the mapping for the details. You can skip the unmatched info.
```

Comment:
Map the file columns to contact properties. You can skip columns that do not match.